app/post-ad/page.tsx

```
@@ -418,6 +584,30 @@ export default function PostAdPage() {
+            <p className="text-muted-foreground">
+              Please wait while your payment is being processed.
+            </p>
+          </div>
+        </DialogContent>
+      </Dialog>
+
+      <Dialog open={isPublishingListing} onOpenChange={setIsPublishingListing}>
+        <DialogContent className="sm:max-w-[425px]">
+          <DialogTitle>Publishing Ad...</DialogTitle>
+          <div className="flex flex-col items-center justify-center py-8">
+            <div className="animate-spin rounded-full h-12 w-12 border-t-2 border-b-2 border-primary mb-4"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)